quill-finagle-postgres/src/main/scala/io/getquill/context/finagle/postgres/FinaglePostgresEncoders.scala

```diff
@@ -11,14 +11,14 @@ import org.jboss.netty.buffer.ChannelBuffers
 trait FinaglePostgresEncoders {
   this: FinaglePostgresContext[_] =>
 
-  protected case class ValueEncoderEncoder[T](encoder: ValueEncoder[T]) extends Encoder[T] {
+  case class ValueEncoderEncoder[T](encoder: ValueEncoder[T]) extends Encoder[T] {
```

Comment:
It needn't, no. I made it public both because the other drivers (Jdbc, Async) do that for their wrappers and in case anyone wanted to use more complex logic or bypass the encoder helper methods in instantiating it, for whatever reason.